taichi/ui/backends/vulkan/renderables/kernels.cu

```diff
@@ -0,0 +1,253 @@
+#include <device_launch_parameters.h>
+#include "taichi/ui/backends/vulkan/vertex.h"
+
+namespace taichi {
+namespace ui {
+
+int div_up(int a, int b) {
```

Comment:
Ah. Didn't know this existed. Although I was trying to avoid including excessive headers into this .cu because it sometimes triggers weird compiler errors and warnings.